### PR TITLE
Docs: update releasing

### PR DESCRIPTION
**Pull request by humitos**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -227,6 +227,7 @@ Releasing the theme
 
 To release a new version of the theme, core team will take the following steps:
 
+#. Install the required depedencies with ``pip install bump2version wheel twine``.
```
Comment (agjohnson) [inline docs/contributing.rst:230]: We already have these defined in the package configuration as well:

https://github.com/readthedocs/sphinx_rtd_theme/blob/68a89bb07e71fbe222718f680d581dfcd0359769/setup.cfg#L55-L59

Twine should be there though.

Comment (humitos) [reply docs/contributing.rst]: Done in e09da77

File: setup.cfg [hunk 2]
```diff
@@ -57,6 +57,7 @@ dev =
 	sphinxcontrib-httpdomain
 	bump2version
 	wheel
+  twine
```
Comment (agjohnson) [inline setup.cfg:60]: I don't think indentation is syntactic in cfg files, but in case it is, the indentation is different here.

**Review by agjohnson — APPROVED** [review]

File: setup.cfg (hunk 2 above)
Comment (humitos) [reply setup.cfg:60]: Meh, tabs vs. spaces 🔥

Comment (agjohnson) [reply setup.cfg:60]: Speaking of bump2version, I think it rewrites this file each time, converting whitespace

**Pull request merged**